.github/workflows/terraform-static-analysis.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       with:
         scan_type: single
         tfsec_exclude: AWS089, AWS099, AWS009, AWS097, AWS018
-        checkov_exclude: CKV_GIT_1
+        checkov_exclude: CKV_GIT_1,CKV_AWS_289,CKV_AWS_290,CKV_AWS_288,CKV_AWS_249,CKV2_GHA_1
```

Comment:
These checks were excluded as fixing them is out of the scope of this PR for getting the tests to pass before @georgepstaylor and I start making our changes going forward - they will be revisited later

Plus the module is already in use with these rules being violated

Reply:
By preference here, those excludes should be scoped to the resources with a justification, rather than a blanket exclusion. Also, `CKV2_GHA_1` is resolvable with correct token permissions.